src/pages/Subscriptions/SubscriptionsMonthlyGood.vue

```diff
@@ -134,9 +249,27 @@ export default {
 				this.dayOfMonth = _get(data, 'my.autoDeposit.dayOfMonth');
 				this.category = _get(data, 'my.monthlyGoodCategory', '');
 				this.mgAmount = autoDepositAmount - this.donation;
+				this.paymentMethod = _get(data, 'my.autoDeposit.paymentMethod', {});
 			}
+
+			// Braintree drop-in UI data
+			const braintreeDropInExp = this.apollo.readFragment({
+				id: 'Experiment:braintree_dropin_mg_update',
+				fragment: experimentVersionFragment,
+			}) || {};
+
+			// if experiment and feature flag are BOTH on, show UI
+			const braintreeDropInFeatureFlag = _get(data, 'general.braintreeDropInFeature.value') === 'true' || false;
+			this.showDropInPaymentUpdate = braintreeDropInFeatureFlag && braintreeDropInExp.version === 'shown';
 		},
 	},
+	mounted() {
+		// Check route params (string) for payment method reset case
+		if (this.$route.query.updatePayment === 'true' && this.showDropInPaymentUpdate) {
```

Comment:
this won't be used to begin with and will require some additional work to enable, but it's ok to keep in here for now.